client/src/notificatonBus.ts

```diff
@@ -0,0 +1,84 @@
+import { RestClient } from '@girder/components/src';
+
+export interface GirderNotification {
+  _id: string;
+  type: string;
+  updated: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
+}
+
+
+/**
+ * Based on Girder Web Components NotificationBus, but simpler.
+ * Register notifications directly on the girderRest instance using
+ * the EventSource api.
+ *
+ * @param rc Girder RestClient
+ */
+export default function registerNotifications(rc: RestClient) {
+  const ES = window.EventSource;
+  const withCredentials = true;
+  const timeoutSeconds = 300;
+  const retryMsDefault = 8_000;
```

Comment:
Underscores are ignored in js numbers.  I use them to signify orders of magnitude.  This is 8 seconds. I originally had it at 10 seconds, so the underscore is particularly nice for readability there

Reply:
Thank You for the info.  I didn't know and did a quick search but didn't see anything immediately in the first page (mostly because my google terms were using underscore so I got a lot of underscore.js results).  Now adjusting the terms I see the info.